src/NServiceBus.Core.Tests/DelayedDelivery/RouteDeferredMessageToTimeoutManagerBehaviorTests.cs

```diff
@@ -71,12 +84,16 @@ public void Cannot_be_combined_with_time_to_be_received()
         {
             var behavior = new RouteDeferredMessageToTimeoutManagerBehavior("tm");
             var delay = TimeSpan.FromDays(1);
-
-            var message = new OutgoingMessage("id", new Dictionary<string, string>(), new byte[0]);
-
+            
             var headers = new Dictionary<string, string>();
-            var context = new RoutingContext(message, new UnicastRoutingStrategy("target"), null);
-            context.AddDeliveryConstraint(new DelayDeliveryWith(delay));
+            var context = new TestableRoutingContext
```

Comment:
in situations like this it would probably make sense to introduce a local "helper" method to reduce the noise
e.g. `CreateRoutingContext(...)`